internal/abci/abci.go

```diff
@@ -228,7 +228,7 @@ func (a *AbciApp) FinalizeBlock(ctx context.Context, req *abciTypes.RequestFinal
 		return nil, fmt.Errorf("begin tx commit failed: %w", err)
 	}
 
-	initialValidators, err := a.txApp.GetValidators(ctx)
+	initialValidators, err := a.txApp.ConsensusValidators(ctx)
```

Comment:
Please see revised `TxApp interface`: https://github.com/kwilteam/kwil-db/pull/768/files#diff-933cda2247bf956073a324e8191848e3f62f7a8a5aa984a13d43933db7345c4aR39

and the docs on the actual methods e.g. https://github.com/kwilteam/kwil-db/pull/768/files#diff-29b2bc6069630247b4e88988ee7f8c5a0c4fa090867fc9d1e5d050c64da892d1R299

Short of merging AbciApp and TxApp, or bringing DB into the purview of AbciApp, we just need to solve a couple things with docs.